packages/integration/src/gitlab/core.ts

```diff
@@ -66,27 +66,34 @@ export function getGitLabRequestOptions(config: GitLabIntegrationConfig): {
 }
 
 // Converts
-// from: https://gitlab.example.com/a/b/blob/master/c.yaml
-// to:   https://gitlab.example.com/a/b/raw/master/c.yaml
+// from: https://gitlab.example.com/groupA/teams/repoA/blob/master/c.yaml
+// to:   https://gitlab.example.com/groupA/teams/repoA/raw/master/c.yaml
 export function buildRawUrl(target: string): URL {
   try {
     const url = new URL(target);
 
-    const [empty, userOrOrg, repoName, blobKeyword, ...restOfPath] =
-      url.pathname.split('/');
+    const splittedPath = url.pathname.split('/');
```

Comment:
```suggestion
    const splitPath = url.pathname.split('/').filter(Boolean);
```

Sorry but "splitted" -> "split" :)

while the Boolean filter is a bit more "lax", it's also really practical - it gets rid of needing to think about the leading `empty` dummy and `v === ''` checks below or whether there's a leading slash or not. Note that it also means you'll need to update the blobIndex range check below.

Reply:
I did it, but if we do the filtering here, we lose empty paths, for example: `gitlab.com/some//repository/blob/some.txt` in this way does not give an error, and the returned URL will be: `gitlab.com/some/repository/raw/some.txt`.
If it is ok for you, we can mantain the filter.